README.md

```diff
@@ -71,15 +72,17 @@ let ``close the account and withdraw the remaining amount`` () =
 ```
 
 There are two kinds of them:
-1. Test what is done (mandatory).\
-   We don't mind how we come up with the outcome, except the outcome has to be correct under the given condition.
-2. Test how it is done (optional).\
-   We are not concerned about the outcome. The only thing needed is to build the state in a particular way.
+1. Test what has been done (mandatory).
+   - We don't mind how we come up with the outcome. 
+   - But, we do need to make sure that the outcome has to be correct under the given condition.
+2. Test how it has been done (optional).
+   - We aren't too concerned about the outcome. 
+   - But, we need to build the state in a particular way.
 
 ### Decider Structure
 
 It's possible to organize the Decider into five sections.\
-So it will be easier to split it into separate files if it starts to grow.
+So it will make it easier to split the implementation into the relevant separate files, especially once it starts to get a little too big.
```

Comment:
@akhansari Not intending to shoot the messenger (great to see this @natalie-o-perret) but I think this should be toned down so people don't willy nilly:
a) break that encapsulation that an [aggregate module](https://github.com/jet/dotnet-templates#aggregate-module) brings
b) follow the dogma that other languages bring - its DDD-ES so CQRS and hundreds of files is normal.

Keep it in one file until it hurts and then decide the best split(s) at the last responsible moment.

(I've seen multiple people coming from backgrounds with thousands of brace-riddled files wanting the comfort of home doing this splitting along technical lines - no encouragement is needed for people to consider this possibility)

Reply:
Yes agree too. Wdyt about:

> It's possible to organize the Decider into five sections.
So it will make it easier to split the implementation into the relevant separate files later.
But at the beginning keep it in one file until it hurts and then decide the best split(s) at the last responsible moment.

Reply:
Are we trying to make ES dummy-proof?🤔

Reply:
Hm, only looked at the table body now - I think having example filenames etc is going too far
For me, being able to `*` and `#` (Vim keys, but Ctrl-F3 etc too) within the file is pretty important 
If you do them as nested modules, in most IDEs, its easy to collapse and then nav to a relevant section.

Based on that I'd express it in terms of groupings of functions

Regarding validation being on the list I would tend to group those alongside the associated decider in the tree.

I agree that the decider functions section frequently benefits from splitting into multiple modules as a way of grouping them (which also encourages defining result types close to where they are needed)
My main objection is that for me its very rare that a given aggregate module grows to this size, whereas this presents it as a critical thing to do.
I guess the best way to answer this is by having more aggregates following the same structure.

Reply:
> I guess the best way to answer this is by having more aggregates following the same structure.
💯